### PR TITLE
F/wpc 611 publish panel notifications

### PR DESCRIPTION
**Pull request by camilleory**
- This PR completes [this one](https://github.com/wepublish/wepublish/pull/493)
- It adds a warning message when the update date is older than the publication date
- It disables the publish panel button when the update date is older than the publication date
- it adds a info message about what the  "available online from" (`publishAt`) does

Jan-27-2022 09-21-37

### CODE REVIEW

File: packages/editor/src/client/atoms/dateTimePicker.tsx [hunk 1]
```diff
@@ -77,7 +88,24 @@ export function DateTimePicker({
   }
   return (
     <>
-      <ControlLabel style={{display: 'block', marginTop: '5px'}}>{label}</ControlLabel>
+      {!helpInfo ? (
+        <ControlLabel style={{display: 'block', marginTop: '5px'}}>{label}</ControlLabel>
```
Comment (tomaszdurka) [inline packages/editor/src/client/atoms/dateTimePicker.tsx:92]: Would be nice to not repeat the control element and just add the Whisper.

**Review by tomaszdurka — CHANGES_REQUESTED** [review]

File: packages/editor/src/client/atoms/dateTimePicker.tsx [hunk 2]
```diff
@@ -19,6 +28,7 @@ export interface DateTimePickerProps {
 
   dateRanges?: DateTimePreset[]
   timeRanges?: DateTimePreset[]
+  helpInfo?: string
```
Comment (penk85) [inline packages/editor/src/client/atoms/dateTimePicker.tsx:31]: Can this be added external to the datePicker component? I'm not sure if it's necessary to be part of this, or would it make more sense to have the info label as part of the publish panel? Alternatively maybe the label can be set as children to allow to pass the label text with any custom icons inside. Not sure if this is possible.

File: packages/editor/src/client/locales/en.json [hunk 3]
```diff
@@ -618,6 +620,7 @@
         "close": "Close",
         "confirm": "Confirm",
         "createdAt": "{{createdAt, ccc PP p}}",
+        "dateExplanationPopOver": "This message will explain precisely what the 'available online from' date does.",
```
Comment (penk85) [inline packages/editor/src/client/locales/en.json:623]: Can we change this? :smile:

Comment (camilleory) [reply packages/editor/src/client/locales/en.json:623]: I'm waiting for Nina's update, she'll write a non-technical sentence about this

Comment (penk85) [inline packages/editor/src/client/locales/en.json:623]: You still need to update this translation from pages. Otherwise looks great. :+1:

Comment (camilleory) [reply packages/editor/src/client/locales/en.json]: Done 👍

**Review by penk85 — APPROVED** [review]

**Review by roosnic1 — APPROVED** [review]
👍🏾

**Review by tomaszdurka — APPROVED** [review]

**Pull request merged**